src/scripts/angry-tock.js

```diff
@@ -27,12 +25,19 @@ const HAPPY_TOCK_REPORT_TO = (process.env.HAPPY_TOCK_REPORT_TO || "#18f").split(
   ","
 );
 
+const dateFromZonedDateTime = (zonedDateTime) =>
+  new Date(Date.parse(zonedDateTime.toInstant()));
+
 /**
  * Get the current time in the configured timezone.
- * @returns {Moment} A moment object representing the current time in the
- *   configured timezone.
+ * @returns {Temporal.ZonedDateTime} A Temporal object representing the current
+ *   time in the configured timezone.
  */
-const m = () => moment.tz(ANGRY_TOCK_TIMEZONE);
+const m = () =>
```

Comment:
Maybe rename this to `t` or even something verbose like `currentZonedDateTime()`?

```suggestion
const currenetZonedDateTime = () =>
```

Reply:
Yes! I put this on my mental checklist of things to come back and do and then forgot. But your comment also reminded me that I'd made this functionality available from the dates utility module, so I switched to using that. 😄